src/main/Java/AnagramKata.java

```diff
@@ -0,0 +1,50 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class AnagramKata {
+    public static Scanner s;
+
+    public AnagramKata(String path) throws FileNotFoundException {
+        File wordList = new File(path);
+        s = new Scanner(wordList);
+    }
+
+    public Integer returnsNumberOfAnagrams(String word) throws FileNotFoundException {
+        Integer totalNumber = 0;
+        while (s.hasNext()) {
+            totalNumber += checksWordsForAnagram(s.next(), word);
+        }
+        s.close();
```

Comment:
We also talked about issues around closeable objects and resource leaks, see if you can refactor this to be safer.